simtools/corsika_simtel/corsika_simtel_runner.py

```diff
@@ -0,0 +1,202 @@
+import stat
+from pathlib import Path
+
+from simtools.corsika.corsika_runner import CorsikaRunner
+from simtools.simtel.simtel_runner_array import SimtelRunnerArray
+
+__all__ = ["CorsikaSimtelRunner"]
+
+
+class MissingRequiredEntryInCorsikaConfig(Exception):
+    """Exception for missing required entry in corsika config."""
+
+
+class CorsikaSimtelRunner(CorsikaRunner, SimtelRunnerArray):
+    """
+    CorsikaSimtelRunner is responsible for running CORSIKA and piping it to sim_telarray
+    using the multipipe functionality. CORSIKA is set up using corsika_autoinputs program
+    provided by the sim_telarray package. It creates the multipipe script and sim_telarray command
+    corresponding to the requested configuration.
+
+    It uses CorsikaConfig to manage the CORSIKA configuration and SimtelRunnerArray
+    for the sim_telarray configuration. User parameters must be given by the
+    common_args, corsika_args and simtel_args arguments.
+    The corsika_args and simtel_args are explained in
+    CorsikaRunner and SimtelRunnerArray respectively.
+    An example of the common_args is given below.
+
+    .. code-block:: python
+
+        common_args = {
+            'label': 'test-production',
+            'simtel_source_path': '/workdir/sim_telarray/',
+        }
+
+    Parameters
+    ----------
+    common_args: dict
+        Arguments common to both CORSIKA and sim_telarray runners
+    corsika_args: dict
+        Arguments for the CORSIKA runner (see full list in CorsikaRunner documentation).
+    simtel_args: dict
+        Arguments for the sim_telarray runner (see full list in SimtelRunnerArray documentation).
+    """
+
+    def __init__(self, common_args=None, corsika_args=None, simtel_args=None):
+
+        CorsikaRunner.__init__(self, use_multipipe=True, **(common_args | corsika_args))
+        SimtelRunnerArray.__init__(self, **(common_args | simtel_args))
+
+    def get_run_script(self, **kwargs):
+        """
+        Get the full path of the run script file for a given run number.
+
```

Comment:
Do I understand it correctly that this function is more than a getter, as it calls the export functions and writes files? Suggest to use a different names, as getters (at least in my understanding) should simply get something. Maybe `prepare_run_script`?

Reply:
Yes, `prepare_run_script` is better so I changed it everywhere to that.